alpha/lib/model/accessControl.php

```diff
@@ -309,4 +330,38 @@ public function getCacheInvalidationKeys()
 	{
 		return array("accessControl:id=".strtolower($this->getId()));
 	}
+
+	public function getHasRemoteServer()
```

Comment:
The function name is not that suitable, should be something like getHasServeRemoteServerRule same applies to other method signature as well.